### PR TITLE
Add desktop breaking change note

### PR DESCRIPTION
**Pull request by amyblais**

### CODE REVIEW

File: source/install/desktop-app-changelog.rst [hunk 1]
```diff
@@ -228,6 +228,7 @@ Compatibility
 ~~~~~~~~~~~~~~~
 
 - Desktop App is supported on any supported Extended Support Release or a newer Mattermost server version.
+- Desktop App v5.2 is incompatible with server versions v9.1 and later.
```
Comment (marianunez) [inline source/install/desktop-app-changelog.rst:231]: Seems that `Desktop App is supported on any supported Extended Support Release` will stop being true after the next 9.x one so wondering if we should also include 
```suggestion
- Desktop App is supported on any supported Extended Support Release up to v8.1 ESR.
- Desktop App v5.2 is incompatible with server versions v9.1 and later.
```

**Review by devinbinnie — APPROVED** [review]

**Review by cwarnermm — APPROVED** [review]

File: source/upgrade/important-upgrade-notes.rst [hunk 2]
```diff
@@ -43,6 +43,8 @@ Important Upgrade Notes
 |                                                    |                                                                                                                                                                  | 
 |                                                    | Added a new configuration setting ``DataRetentionSettings.RetentionIdsBatchSize``, which allows admins to configure how many batches of IDs will be fetched at   |
 |                                                    | a time when deleting orphaned reactions. The default value is 100.                                                                                               |
+|                                                    +------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+|                                                    | Modifield OAuth/SAML flows to include ``desktop_login`` where applicable. This change makes Desktop v5.2 and below incompatible with server v9.1 and later.      |
```
Comment (jasonblais) [inline source/upgrade/important-upgrade-notes.rst:47]: To confirm, the server is compatible on v5.3 and later?

**Review by jasonblais — COMMENTED** [review]
What is our policy for which desktop app versions we officially support?

**Comment by amyblais** [comment]
> What is our policy for which desktop app versions we officially support?

That's under discussion https://community.mattermost.com/private-core/pl/qkwkt4ru8iygt8pmt4ns9u5wfw.

File: source/upgrade/important-upgrade-notes.rst (hunk 2 above)
Comment (amyblais) [reply source/upgrade/important-upgrade-notes.rst:47]: Correct

File: source/install/self-managed-changelog.md [hunk 3]
```diff
@@ -18,6 +18,7 @@ Latest Mattermost Releases:
 
 ### Important Upgrade Notes
  - Improved performance on data retention ``DeleteOrphanedRows`` queries. See the [Important Upgrade Notes](https://docs.mattermost.com/upgrade/important-upgrade-notes.html) for notes on a new migration that was added. Removed feature flag ``DataRetentionConcurrencyEnabled``. Data retention now runs without concurrency in order to avoid any performance degradation. Added a new configuration setting ``DataRetentionSettings.RetentionIdsBatchSize``, which allows admins to to configure how many batches of IDs will be fetched at a time when deleting orphaned reactions. The default value is 100.
+ - Modifield OAuth/SAML flows to include ``desktop_login`` where applicable. This change makes Desktop v5.2 and below incompatible with server v9.1 and later.
```
Comment (jasonblais) [inline source/install/self-managed-changelog.md:21]:
```suggestion
 - Minimum supported Desktop App version is now v5.3. OAuth/SAML flows were modified to include ``desktop_login`` which makes earlier versions incompatible.
```

File: source/upgrade/important-upgrade-notes.rst (hunk 2 above)
Comment (jasonblais) [inline source/upgrade/important-upgrade-notes.rst:47]:
```suggestion
|                                                    | Minimum supported Desktop App version is now v5.3. OAuth/SAML flows were modified to include ``desktop_login`` which makes earlier versions incompatible.     |
```

**Review by jasonblais — CHANGES_REQUESTED** [review]
Proposed suggestions. Should this also be included in the Cloud changelog?

**Review by jasonblais — APPROVED** [review]

**Pull request merged**